src/Stripe.Tests/transfers/test_data/stripe_transfer_create_options.cs

```diff
@@ -5,13 +5,13 @@ namespace Stripe.Tests.test_data
 {
     public class stripe_transfer_create_options
     {
-        public static StripeTransferCreateOptions Valid()
+        public static StripeTransferCreateOptions Valid(string recipient = "self")
```

Comment:
What are the other options here? This should probably be a static class in the Types folder to follow the convention.